doc/index.rst

```diff
@@ -18,12 +18,13 @@ Luma.LCD: Display drivers for PCD8544, ST7735, ST7567, HT1621, UC1701X, ILI9341
    :maxdepth: 2
 
    intro
-   install
```

Comment:
having install as high up in list as possible has my preference.

Reply:
Moved install up to the second section (right after intro).  The only issue is need to make sure that the hardware section requisites have been completed but no harm in having hardware a bit later.